main.go

```diff
@@ -1,21 +1,73 @@
 package main
 
 import (
+	"encoding/json"
 	"fmt"
+	"io/ioutil"
 	"os"
+	"os/user"
+	"path/filepath"
 
 	"github.com/abs-lang/abs/repl"
+	"github.com/hashicorp/go-getter"
 )
 
 var Version = "1.8.0"
 
+func check(e error) {
+	if e != nil {
+		panic(e)
+	}
+}
+
 // The ABS interpreter
 func main() {
 	args := os.Args
 	if len(args) == 2 && args[1] == "--version" {
 		fmt.Println(Version)
 		return
 	}
+
+	if args[1] == "--get" {
```

Comment:
can we use `get` and not `--get`? npm, go, etc all use a command without the `--` (else it's a flag). I'm anyhow planning to phase out `--version` and replace it with `version` in abs 2.